flink-connectors/pom.xml

```diff
@@ -105,6 +105,7 @@ under the License.
 				<module>flink-sql-connector-hive-3.1.2</module>
 				<module>flink-sql-connector-kafka</module>
 				<module>flink-sql-connector-kinesis</module>
+				<module>flink-sql-connector-pulsar</module>
```

Comment:
Pulsar connector doesn't support the Flink SQL now. Add this module may cause confusion for the end-user. Can we just use `flink-connector-pulsar`?

Reply:
The `flink-sql-connector-pulsar` is a uber jar is used by PyFlink. Inspired by `flink-connector-rabbitmq` and ` flink-sql-connector-rabbitmq`.